homeassistant/components/vicare/climate.py

```diff
@@ -169,6 +169,12 @@ def update(self):
                 self._attributes[
                     "return_temperature"
                 ] = self._api.getReturnTemperature()
+                self._attributes[
+                    "return_temperature_primary_circuit"
```

Comment:
What exactly is the `return_temperature_primary_circuit`? Wouldn't it be better to just call it `temperature_primary_circuit`?